src/views/Vote/Vote.vue

```diff
@@ -63,6 +76,119 @@ export default class Vote extends Vue {
     };
     const voting = await this.getVoting(input);
     this.$emit("set-subtitle", formatToDate(voting.createdAtDateTime));
+    this.isLoaded = true;
+    this.animateScore(this.voteScore, true);
+    window.addEventListener("resize", () => {
+      this.isLandscape();
+    });
+  }
+
+  public isLandscape() {
+    if (window.innerHeight > window.innerWidth) {
```

Comment:
landscape is when window width is greater than window's height right ? Wouldn't be simpler if you simply put:
```typescript
this.landscape = window.innerWidth > window.innerHeight
```